libs/auth/src/openssl.ts

```diff
@@ -186,13 +188,17 @@ export async function verifySignature({
  */
 export async function createCert({
   certSubject,
+  certType = 'standard',
+  expiryInDays = 365,
```

Comment:
Working on this script reminded me that we haven't discussed card cert expiry - how long should card certs last before expiring?

Sys admin card certs should probably last longer than election manager and poll worker card certs.

Reply:
Bubbled this question up to Slack since it has product implications. Let's discuss there and I'll document the result here!

Reply:
Landed on the following:
1. 100 years for card VotingWorks certs
2. 5 years for sys admin card VxAdmin certs
3. 6 months for election manager and poll worker card VxAdmin certs

Will open a separate PR for this since it's all fairly tangential to this PR.

For 2, we have a future optimization in mind where we decrease the sys admin card VxAdmin cert expiry to 1 year but auto-refresh the expiry every time the card is used. Opened an issue to track this: https://github.com/votingworks/vxsuite/issues/3109

Reply:
Good call! I'll incorporate this into my PR for https://github.com/votingworks/vxsuite/issues/3110